### ISSUE TITLE
[Feature Request] Select which components to install

### ISSUE THREAD

**Issue by d8ahazard** [cjmurph/PmsService#52]
This is more for my own reference, but just so it's out there for discussion:

I think it would be a smashing idea to adjust the installer so that the user can select which apps to install.

I honestly didn't even consider that the tray app could be used on a separate machine until it was mentioned when I broke the log stuff.

But, it's an awesome feature, and I think more people would be inclined to use it if they had the option to only install the tray component.

**Comment by cjmurph** [comment]
Yes, definetly. I think this was maybe the first wix installer I ever wrote. It could do with some love splitting the install up into selectable features.
